### PR TITLE
Add statistics commands

### PR DESCRIPTION
**Pull request by Frojdholm**
Right now the module ignores bot and commands. This has the side-effect that users can "hide" their messages from the logging by doing `?valid_command <messages here>`.

### CODE REVIEW

File: src/globals.py [hunk 1]
```diff
@@ -9,10 +9,14 @@
     CFG = json.load(data_file)
 
 # Temporarily stored as a global. In the future this will change depending on the environment
-DATABASE = MySQLDatabase(database='aryas',
-                         user=CFG["aryas"]["db"]["user"],
-                         password=CFG["aryas"]["db"]["pass"],
-                         host=CFG["aryas"]["db"]["host"])
+# DATABASE = MySQLDatabase(database='aryas',
```
Comment (tom-sherman) [inline src/globals.py]: Change this back to MySQL

File: src/utility.py [hunk 2]
```diff
@@ -5,6 +5,12 @@
 from src.globals import MESSAGE_SLEEP_TIME, logger, MOD_LOG_CHANNEL_NAME
 
 
+def is_command(bot, cmd):
```
Comment (tom-sherman) [inline src/utility.py:8]: Add docstring please.

File: src/extensions/statistics.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+        embed = discord.Embed(color=discord.Color(COLOR), timestamp=datetime.datetime.now())
+        embed.set_footer(text='Global footer for all embeds', icon_url='https://cdn.discordapp.com/embed/avatars/2.png')
+        for user in users:
+            embed.add_field(name=user.name, value='Total messages: {}'.format(user.total_messages))
+        await self.bot.say(content='Top active users:', embed=embed)
+
+
+def setup(bot: commands.Bot):
+    bot.add_cog(Statistics(bot))
```
Comment (tom-sherman) [inline src/extensions/statistics.py]: Newline at EOF pls

**Review by tom-sherman — CHANGES_REQUESTED** [review]
Few ez changes

File: src/utility.py (hunk 2 above)
Comment (Frojdholm) [reply src/utility.py:8]: Fixed, but this function should maybe be removed

File: src/extensions/statistics.py (hunk 3 above)
Comment (Frojdholm) [reply src/extensions/statistics.py]: fixed

**Pull request closed without merging**